ray_on_golem/server/services/yagna.py

```diff
@@ -65,6 +66,7 @@ async def _run_yagna_service(self) -> None:
             "run",
             stdout=asyncio.subprocess.DEVNULL,
             stderr=asyncio.subprocess.DEVNULL,
```

Comment:
Can we still access yagna logs soemhow?

Reply:
Yes, `tail -n 50 -f ~/.local/share/yagna/yagna_rCURRENT.log`